### DISCUSSION TITLE
Print option, Full Screen, responsive, download

### DISCUSSION THREAD

**Discussion by ahmerarain** [KevinVandy/mantine-react-table#100] [Ideas]
- there should be an option to print and according to select print paper size table should adjust accordingly 
- on full screen, auto pagination off props
- mobile responsive, like in mobile its shows each row as a separate card/div 
- should the option to download  as xl/CSV
-

**Comment by KevinVandy** [comment]
All these things are possible in MRT, just not built in and never will be
